### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by frederfred**
Let the documentation begin!

### CODE REVIEW

**Comment by frederfred** [comment]
@jsahlen I'm thinking we can merge this (if you don't spot any errors) and then talk about what to keep, move and remove. And after that continue to document the rest.

File: lib/repetitivo/functions/strip-units.scss [hunk 1]
```diff
@@ -1,8 +1,8 @@
-// Strip units from a number
-// @param {Number} $number The number to strip
-// @return {Number}
-// @example
-//   strip-units(4px) // 4
+/// Strip units from a number
+/// @param {number} $number The number to strip
```
Comment (jsahlen) [inline lib/repetitivo/functions/strip-units.scss]: There should be a dash between `$number` and `The number to strip`, right?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,10 @@
 # repetitivo
+
 A system.
+
+## Documentation
+
+Build documentation to `docs` with `npm run generate:docs`.
```
Comment (jsahlen) [inline README.md]: Maybe add a slash to clarify that it's a directory? I.e. `docs/`

File: lib/repetitivo/functions/strip-units.scss (hunk 1 above)
Comment (frederfred) [reply lib/repetitivo/functions/strip-units.scss]: Yes, good catch!

File: README.md (hunk 2 above)
Comment (frederfred) [reply README.md]: Good suggestion.

**Pull request merged**